msteams-platform/bots/how-to/conversations/bot-apis-with-rsc.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Receive all channel messages with RSC
+author: surbhigupta12
+description: Receive all channel messages with RSC permissions
+ms.topic: conceptual
+localization_priority: Normal
+---
+
+# Receive all channel messages with RSC
+
+> [!NOTE]
+> This feature is available in public developer preview.
+
+The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is now extended to bot scenarios.
+
+Currently, bots can only receive user channel messages when they are @mentioned. Using RSC, you can now request team owners to consent for a bot to receive user messages across all channels in a team without using @mention. This capability is enabled by specifying the `ChannelMessage.Read.Group` permission in the manifest of an RSC enabled Teams app. After configuration, Team owners can grant consent during the app installation process.
```

Comment:
> receive user messages across all channels

Please specify "standard channels". This feature does not apply to private channels

> in a team without using @mention

in a team without ~~using~~ being @mentioned